pkg/transport/process/extensions/uds_executable.go

```diff
@@ -0,0 +1,118 @@
+// SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors.
+//
+// SPDX-License-Identifier: Apache-2.0
+package extensions
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net"
+	"os"
+	"os/exec"
+	"syscall"
+	"time"
+
+	"github.com/gardener/component-cli/pkg/transport/process"
+	"github.com/gardener/component-cli/pkg/utils"
+)
+
+// ServerAddressEnv is the environment variable key which is used to store the
+// address under which a resource processor server should start.
+const ServerAddressEnv = "SERVER_ADDRESS"
```

Comment:
use a domain-like name to avoid clashes with preexisting env variables with this name?

Reply:
improved the naming, but wouldn't use domain like scheme because these executables are started by our tool with a clean env (no clashes) and any additional env variable must be explicitely declared by the user

Reply:
ok, valid point.